tests/integration_python/test_run_cli.py

```diff
@@ -1123,3 +1120,31 @@ def test_short_circuit_composition(pixi: Path, tmp_pixi_workspace: Path) -> None
 
     assert output1.stdout == output2.stdout
     assert output1.stderr == output2.stderr
+
+
+def test_task_minijinja_title_filter(pixi: Path, tmp_pixi_workspace: Path) -> None:
```

Comment:
Let's add a more significant set of tests here, as this might be pretty vulnerable to updates and small changes. 

Let's at least also test multiple arguments, the template syntax and without arguments.